test.sh

```diff
@@ -35,9 +35,8 @@ do
 	fi
 done
 echo "Main: ./test/Tests.ooc" >> "$TESTS_USE_FILE"
-rm .libs/tests-linux64.*
-rock -q -lpthread --gc=off $ARGS $FLAGS $TESTS_USE_FILE 2>&1
```

Comment:
It was needed in the old script to suppress errors but I believe we don't need it anymore.